lib/fined.js

```diff
@@ -0,0 +1,141 @@
+const fs = require('fs');
+const path = require('path');
+
+const isArray = require('lodash.isarray');
+const isString = require('lodash.isstring');
+const isPlainObject = require('lodash.isplainobject');
+const isEmpty = require('lodash.isempty');
+const mapValues = require('lodash.mapvalues');
+const pick = require('lodash.pick');
+
+const expandTilde = require('expand-tilde');
+const parsePath = require('parse-filepath');
+
+
+function search(options, extensions, cwd) {
```

Comment:
@tkellen I've understood now that a single file you say is really a single file to be found by `fined` at a time. I misunderstood that it was a single file by each properties of `configFiles` and `fined` found files for all properties at a time. I'm sorry.